src/sync/branches.ts

```diff
@@ -6,25 +6,28 @@ import { BackfillMessagePayload } from "~/src/sqs/sqs.types";
 
 // TODO: better typings
 export const getBranchTask = async (
-	logger: Logger,
+	parentLogger: Logger,
 	gitHubClient: GitHubInstallationClient,
 	_jiraHost: string,
 	repository: Repository,
 	cursor: string | undefined,
 	perPage: number,
 	messagePayload: BackfillMessagePayload) => {
 
-	logger.info("Syncing branches: started");
+	const logger = parentLogger.child({ backfillTask: "Branch" });
+	const startTime = Date.now();
+
+	logger.info({ startTime }, "Backfill task started");
```

Comment:
Put it as `Branch backfill task started`? As if just `Backfill task started` it is a bit confusing.
And also I don't think we need this line, as we should have already something in a upper level? In the coordinator of all the tasks? So instead of doing it in each, leave it to parent?